File: Fizz_Buzz/fizz-buzz.js
// Return an array
function fizzbuzz(n){
	let array = [];
	for (i = 1; i< n+1; i++){
		if (i%3 == 0 && i%5 == 0){ 
		array[i-1] = "FizzBuzz"
		}else if (i%5 == 0){
			array[i-1] ="Buzz"

		}else if( i%3 == 0){
			array[i-1] ="Fizz"
		}else{
			array[i-1] =i
		}
	}; 
	return array 
}

console.log(fizzbuzz(15))

File: Fizz_Buzz/fizz-buzz.py
#standard for loop

def fizzbuzz(n):
    array = list(range(1, n+1))
    for i in array: 
        if i%3 ==0 and i%5 == 0: 
            array[i-1] = "FizzBuzz"
        elif i%5 == 0: 
            array[i-1] = "Buzz"
        elif i%3 == 0: 
            array [i-1] = "Fizz"
    return array


print(fizzbuzz(30))


